Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/ComboBoxInput.swift

```diff
@@ -233,9 +233,11 @@ extension ComboBoxInput {
         }
     }
     
-    private func updateValue(_ value: (any Sendable)?) {
-        element.updateValue(value)
-        internalFeatureFormViewModel.evaluateExpressions()
+    private func updateValueAndEvaluateExpressions(_ value: CodedValue?) {
+        if element.formattedValue != value?.name {
+            element.updateValue(value?.code)
+            internalFeatureFormViewModel.evaluateExpressions()
+        }
```

Comment:
This is a [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions), which we try to avoid. Also, it would make sense to use a guard statement here.

```suggestion
        guard value?.name != element.formattedValue else { return }
        element.updateValue(value?.code)
        internalFeatureFormViewModel.evaluateExpressions()
```